migrations/20220804162029-create-site-meal-type.js

```diff
@@ -10,9 +10,23 @@ module.exports = {
       },
       SiteId: {
         type: Sequelize.INTEGER,
+        allowNull: false,
+        references: {
+          model: {
+            tableName: 'Sites',
+          },
+          key: 'id',
+        },
```

Comment:
This makes sure that a valid id (not null) from the corresponding table is in the column by setting up a FOREIGN KEY relationship in the database table.